src/Configuration/Schema/SchemaConfiguration.php

```diff
@@ -77,12 +91,23 @@ public function __construct(
         Assert::nullOrGreaterThanEq($timeout, 1);
         Assert::nullOrOneOf($testFramework, TestFrameworkTypes::TYPES);
 
+        if ($minMsi !== null) {
+            Assert::true(is_int($minMsi) || is_float($minMsi));
```

Comment:
`is_*` are missing import statements,  note that you can configure it in phpstorm to avoid having this issue all the time. I would also be happy to have it automated but couldn't find a way to do so

Reply:
Can probably be removed now, when we use type declarations!

Reply:
We can probably write our custom PHPStan plugin. I know it would be great to have PHP-CS-Fixer rule instead, but personally I hate working with tokens and won't contribute there, unfortunately.